sql/src/test/java/io/crate/integrationtests/TransportSQLActionSingleNodeTest.java

```diff
@@ -322,6 +323,65 @@ public void run() {
         }
     }
 
+    @Test
+    public void testKilledJobLog() throws Exception {
```

Comment:
this test looks fairly expensive and I don't really understand what it is testing?